website/content/docs/nia/release-notes/0-5-0.mdx

```diff
@@ -0,0 +1,215 @@
+---
+layout: docs
+page_title: CTS v0.5.0
+description: >-
+  Consul-Terraform-Sync Configuration Deprecations for v0.5.0
+---
+
+# Configuration Deprecations in v0.5
+
+We introduced a number of configuration deprecations in Consul-Terraform-Sync (CTS) v0.5. Many of these deprecations are part of an effort to streamline CTS task configuration and reduce the inconsistencies. As CTS supported more use-cases, the inconsistencies grew and required a lot of documentation to explain.
+
+This documentation outlines these deprecations and examples on how to upgrade your configuration for v0.5. The deprecated configurations will not be removed until v0.7 and later. We plan to also release a tool to automate upgrading your configuration to hopefully make the upgrade quicker and smoother.
+
+## Overview of Deprecations
+
+The deprecations attempt to address two major inconsistencies. First is the use of the terminology "source" in various configuration blocks and fields. The second is the exception to configuring a task condition or module input using the `services` field and related its `service` block
```

Comment:
```suggestion
The deprecations address two major inconsistencies. First is the use of the terminology "source" in various configuration blocks and fields. The second is the exception to configuring, a task condition, or module input using the `services` field and related its `service` block
```

Removed the word "attempt". Again, let's be confident in our writing 😄

Reply:
I'm realizing I have some typos in "The second..." sentence which are confusing. Let me know if you have thoughts on:

> The second is the exception to configuring a task condition or a task module input by using the `services` field and its related `service` block..

Reply:
That makes more sense 👍🏼